npc_simulator/include/npc_simulator/node.h

```diff
@@ -185,6 +176,38 @@ class NPCSimulatorNode
   const double max_dist_uturn_ = 10.0;
   const double max_delta_yaw_uturn_ = boost::math::constants::pi<double>() / 3.0;
 
+  template <class T>
+  T waitForParam(const std::string & key)
```

Comment:
For vehicle parameters, please use `vehicle_info_util`. Just make that class a member of your node and initialize it like in node.cpp here: https://github.com/tier4/Pilot.Auto/pull/64/files#diff-802ee24659db98121c936879be0e12cf2e36b95e595239a5f98ccea2cae63d17

Reply:
thanks for the hints, I did it